pkg/asset/installconfig/openstack/validation/cloudinfo.go

```diff
@@ -105,13 +105,6 @@ func (ci *CloudInfo) collectInfo(ic *types.InstallConfig) error {
 	}
 
 	// Get flavor info
-	if flavorName := ic.OpenStack.FlavorName; flavorName != "" {
```

Comment:
Should this be replaced with a fetch of the flavor from the DefaultMachinePlatform?

Reply:
oh, sure... thanks for noticing this!

Reply:
I started validating DefaultMachinePlatform here  https://github.com/openshift/installer/pull/4321/files#diff-99d5288a13cd46e2e89659b42a327f77fcb0c806f651085b9458843e767cf767R32-R34
Flavor will be checked as a part of this validation.